docs/functions/spawnDialog.md

```diff
@@ -11,19 +11,41 @@ import {
 ## Definitions
 
 ```ts static
+type SpawnDialogOptions = {
+	/**
+	 * Container to mount the dialog to
+	 * @default document.body
+	 */
+	container?: Element | string
+}
+
+export function spawnDialog(
+	dialog: Component,
+	props?: object,
+	onClose?: (...rest: unknown[]) => void,
+): void
+
+export function spawnDialog(
+	dialog: Component,
+	props?: object,
+	options?: SpawnDialogOptions,
+	onClose?: (...rest: unknown[]) => void,
+): void
```

Comment:
Especially with this overload I would think that a promise is nicer (too many parameters IMHO)

```suggestion
export function spawnDialog(
	dialog: Component,
	props?: object,
	options?: SpawnDialogOptions,
): Promise<unknown[]>
```

Reply:
Removing `onClose` is a breaking change

(But it's fine to me)

Reply:
https://github.com/nextcloud-libraries/nextcloud-vue/pull/6759